### DISCUSSION TITLE
v9 is shipping an outdated NPoco

### DISCUSSION THREAD

**Discussion by arknu** [umbraco/Umbraco-CMS#10911] [Features and ideas]
I noticed that v9 is still shipping with NPoco v4 (specifically v 4.0.2). Shouldn't this be upgraded to v5 (released November 2020) before releasing? I would argue that taking any breaking changes now would be better than after releasing v9 - though breakage should be pretty minor.
If we are too close for release to upgrade, when will it be updated? In v10?

**Comment by rjsmith2** [comment]
Yes, please!

There are couple of npoco methods that lacks async. They exists in v5 now.

**Comment by p-m-j** [comment]
There's a draft PR here https://github.com/umbraco/Umbraco-CMS/pull/9800 that looks like it is almost complete
